src/lr_apply.rs

```diff
@@ -47,12 +46,8 @@ unsafe fn lr_stripe<BD: BitDepth>(
     let sby = y + (if y != 0 { 8 << ss_ver } else { 0 }) >> 6 - ss_ver + seq_hdr.sb128;
     let have_tt = (c.tc.len() > 1) as c_int;
     let lpf_stride = BD::pxstride(stride);
-    let lpf_plane_sz = BD::pxstride(f.lf.lr_buf_plane_sz[(plane != 0) as usize] as isize);
-    let mut lpf_offset = cmp::max(lpf_stride - lpf_plane_sz, 0);
```

Comment:
@fbossen I ended up removing `lpf_plane_sz` and `lpf_offset` in order to make the calculation here the same as the original C. You had added those in #746 to handle negative strides I think. Do we still need those with the fixed `pxstride`? Was there a reason why we needed those values in Rust but not in the original C?

Reply:
The reason I added `lpf_plane_sz` and `lpf_offset` is that I defined `lpf` as a slice that begins at the memory section being effectively used and needed the compute the memory location of the beginning of the slice (and cover the case of a negative stride). If, instead of defining a new slice `lpf`, you use `lr_line_buf`, then removing `lpf_plane_sz` and `lpf_offset` looks appropriate.